packages/nc-gui/pages/index/index/index.vue

```diff
@@ -175,18 +175,18 @@ const getProjectPrimary = (project: ProjectType) => {
     </div>
 
     <TransitionGroup name="layout" mode="out-in">
-      <div v-if="isLoading">
+      <div v-if="isLoading" key="skeleton">
```

Comment:
Why there is a static key for these elements and any specific reason to keep them?

Reply:
Yeah Vue's `transition-group` requires a key on elements :)